CHANGELOG.md

```diff
@@ -1,4 +1,5 @@
 ### Enhancements
+* [vra] 250 / Add support for global scope property-group.
```

Comment:
I have moved the changelog to archive and changed the PR title to match changelog / release notes standard. Could you double check the merge conflict here and either skip the CHANGELOG entry change or just do it in the archive location